action_server/src/sema4ai/action_server/_cli_impl.py

```diff
@@ -289,6 +289,14 @@ def _add_new_command(command_subparser, defaults):
 
     add_verbose_args(list_templates_parser, defaults)
     add_json_output_args(list_templates_parser)
+    
+    
+def _add_get_default_oauth_config_command(command_subparser, defaults):
+    # This serves only as an internal command, therefore we don't add help entry to it.
+    command_subparser.add_parser(
+        "get-default-oauth-config",
```

Comment:
I guess this is the default sema4.ai config? Should the name be: get-sema4ai-oauth-config?